numpy/distutils/ccompiler.py

```diff
@@ -162,7 +165,7 @@ def CCompiler_spawn(self, cmd, display=None):
     if is_sequence(cmd):
         cmd = ' '.join(list(cmd))
 
-    forward_bytes_to_stdout(o)
+    log.info(o)
```

Comment:
I think there's a problem here that `o` has type `bytes`, but log.info expects `str`. Previously we got around this by assuming we could pass GCC's output directly to the terminal without decoding and re-encoding, which saved us having to guess GCC's encoding.

Reply:
fixing (make it conditional on `self.verbose`)